app/scripts/acs/acs-graphs.js

```diff
@@ -18,8 +18,12 @@
         return module;
 
 
+        /**
+         * Replace text of length > max with an ellipsis
+         * Do no replacement if max === -1
+         */
         function addElipses(text, max) {
-            if (text.length > max) {
+            if (max !== -1 && text.length > max) {
```

Comment:
Never mind, I see. Why not check to see if `max` is truthy and set to `null` in config instead of `-1`?